### PR TITLE
add generate from data sample for CRUDs and SVs

### PR DESCRIPTION
**Pull request by albertotessarotto**
## Pull Request Type

- [ ] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [ ] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [ ] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [ ] No sensitive content has been committed

### CODE REVIEW

File: docs/development_suite/api-console/api-design/crud_advanced.md [hunk 1]
```diff
@@ -353,6 +360,42 @@ If you want to delete a row, select red trash symbol on the right of the the tab
...
+At the end of the upload an internal function will try to cast the types correctly, otherwise it will treat them as strings by default.
+
+:::note
+Import of fields is supported only for the following data types: `String`, `Number`, `Boolean` or `Date`. For example, you cannot import fields of type object.
+In case some fields are not in the correct format, they will be ignored.
+:::
+
+:::caution
+You cannot import fields with the same name as one of the metadata fields. If this happens, a warning will be displayed and the import won't be successful
+:::
```
Comment (albertotessarotto) [inline docs/development_suite/api-console/api-design/crud_advanced.md:397]: @danibix95 are these notes correct?

File: docs/fast_data/configuration/single_views.md [hunk 2]
```diff
@@ -22,7 +22,54 @@ To define the custom fields, indexes and internal endpoints of your single view
...
+At the end of the upload an internal function will try to cast the types correctly, otherwise it will treat them as strings by default.
+
+:::note
+Import of fields is supported only for the following data types: `String`, `Number`, `Boolean` or `Date`. For example, you cannot import fields of type object.
+In Case some fields are not in the correct format, they will be ignored.
+:::
+
+:::caution
+You cannot import fields with the same name as one of the metadata fields. If this happens, a warning will be displayed and the import won't be successful
+:::
```
Comment (albertotessarotto) [inline docs/fast_data/configuration/single_views.md:69]: @danibix95 are these notes correct?

File: docs/development_suite/api-console/api-design/crud_advanced.md [hunk 3]
```diff
@@ -172,9 +172,16 @@ You can always change the default state of an existing route by simply clicking
...
 
-### How to create the fields of your CRUD by importing a JSON
+### Import CRUD collection fields
 
-If you want, you can upload your fields from file, selecting **Import fields from File**.
+Instead of manually creating your CRUD fields, you can easily import them:
+
+- from a [JSON schema](#how-to-create-the-fields-of-your-crud-by-importing-a-json)
+- from a [data sample](#how-to-generate-from-data-sample-your-crud-fields)
+
```
Comment (danibix95) [inline docs/development_suite/api-console/api-design/crud_advanced.md:181]:
```suggestion
In addition to manually manually creating your CRUD fields, it is also possible to import them:

- from a [JSON schema](#how-to-create-the-fields-of-your-crud-by-importing-a-json)
- from a [data sample](#how-to-generate-from-data-sample-your-crud-fields)

This simplify the schema definition whenever it has already been defined somewhere else or a record representative of your dataset is available.
```
What do you think?

File: docs/development_suite/api-console/api-design/crud_advanced.md (hunk 1 above)
Comment (danibix95) [reply docs/development_suite/api-console/api-design/crud_advanced.md:397]: Yep, the behavior is consistent.

File: docs/fast_data/configuration/single_views.md (hunk 2 above)
Comment (danibix95) [reply docs/fast_data/configuration/single_views.md:69]: Same above

File: docs/fast_data/configuration/single_views.md [hunk 4]
```diff
@@ -22,7 +22,54 @@ To define the custom fields, indexes and internal endpoints of your single view
 
 ![Fast Data Single View Fields](../img/single-view-detail-fields.png)
 
-The type fields supported are the same of the collection you can create in the [MongoDB CRUD section](/development_suite/api-console/api-design/crud_advanced.md#create-a-new-crud). To know more about indexes [click here](/runtime_suite/crud-service/10_overview_and_usage.md#indexes).
+### Import Single view fields
+
+Instead of manually creating your single view fields, you can easily import them:
```
Comment (danibix95) [inline docs/fast_data/configuration/single_views.md:27]: Should we apply the same suggestion as above?

File: docs/development_suite/api-console/api-design/crud_advanced.md (hunk 3 above)
Comment (albertotessarotto) [reply docs/development_suite/api-console/api-design/crud_advanced.md:181]: super, thanks

File: docs/development_suite/api-console/api-design/crud_advanced.md [hunk 5]
```diff
@@ -353,6 +361,42 @@ If you want to delete a row, select red trash symbol on the right of the the tab
 The collection **has not yet been saved** it is necessary to continue the process described below
 :::
 
+#### How to generate from data sample your CRUD fields
+
+From the `Import fileds` button, select the **From data sample** option.
```
Comment (albertotessarotto) [inline docs/development_suite/api-console/api-design/crud_advanced.md:366]:
```suggestion
From the `Import fields` button, select the **From data sample** option. 
```

**Pull request merged**